src/main/java/com/passbook/sparkeighteen/peristence/POJO/ProfileRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.passbook.sparkeighteen.peristence.POJO;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProfileRequest {
+    @NotBlank(message = "mobile number is mandatory")
+    private String mobileNumber;
+
+    @NotBlank(message = "address is mandatory")
+    private String address;
+
+    private String pan;
```

Comment:
regex?

Reply:
https://www.incometaxindia.gov.in/Forms/tps/1.Permanent%20Account%20Number%20(PAN).pdf

Reply:
Validation Done, we user proper PAN pattern 

For more understanding
https://www.incometaxindia.gov.in/Forms/tps/1.Permanent%20Account%20Number%20(PAN).pdf